Build failing for microchip's SAMR21G18_MODULE

Hello guys!
I'm trying to build the openthread drivers for the SAMR21G18_MODULE.

**Description**
The build works fine for the SAMR21_XPLAINED_PRO, however, I am compiling drivers for their ATZB-SAMR21-PRO board, which is a wireless extension module for the evaluation kit.

**To Reproduce**
I followed the build instructions mentioned in the microchip platform folder: [microchip_samr21](https://github.com/openthread/openthread/tree/master/examples/platforms/samr21) .
I encountered the error when I ran,
```
make -f examples/Makefile-samr21 BOARD=SAMR21G18_MODULE
```

**Console output**
```
openthread/examples/../examples/platforms/samr21/logging.c:57:13: error: 'logOutput' defined but not used [-Werror=unused-function]
 static void logOutput(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, va_list ap)
             ^~~~~~~~~
cc1: all warnings being treated as errors
Makefile:952: recipe for target 'libopenthread_samr21_a-logging.o' failed
make[5]: *** [libopenthread_samr21_a-logging.o] Error 1
make[5]: *** Waiting for unfinished jobs....
Makefile:472: recipe for target 'all-recursive' failed
make[4]: *** [all-recursive] Error 1
Makefile:438: recipe for target 'all-recursive' failed
make[3]: *** [all-recursive] Error 1
Makefile:594: recipe for target 'all-recursive' failed
make[2]: *** [all-recursive] Error 1
Makefile:520: recipe for target 'all' failed
make[1]: *** [all] Error 2
examples/Makefile-samr21:251: recipe for target 'do-build-cortex-m0plus' failed
make: *** [do-build-cortex-m0plus] Error 2
```

**Bypassing the error**
I am currently trying to disable logging by changing the options in the examples/common-switches.mk file. I don't know if that will help as that function is defined in the top layer, and is not module specific.

**Additional context**
OS: Linux Ubuntu 18.04 LTS
Linux Version: 5.3.0-46-generic
Board: ATZB-SAMR21-PRO (ATSAMR21G18-MR210UA chip)
arm-none-eabi-gcc version: 7.3.1 20180622 (release) [ARM/embedded-7-branch revision 261907]

--- a/examples/platforms/samr21/logging.c
+++ b/examples/platforms/samr21/logging.c
@@ -32,19 +32,19 @@
  *
  */
 
-#include <openthread-core-config.h>
+#include "openthread-core-config.h"
+
 #include <utils/code_utils.h>
-#include <openthread/config.h>
 #include <openthread/platform/logging.h>
 #include <openthread/platform/toolchain.h>
+
+#include "board.h"
+#include "spi.h"
 
 #if (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_PLATFORM_DEFINED) || \
     (OPENTHREAD_CONFIG_LOG_OUTPUT == OPENTHREAD_CONFIG_LOG_OUTPUT_NCP_SPINEL)
 
 #if BOARD == SAMR21_XPLAINED_PRO
-
-#include "board.h"
-#include "spi.h"
 
 #define LOG_PARSE_BUFFER_SIZE 128
 #define LOG_TIMESTAMP_ENABLE 1

